cfn-resources/teams/cmd/resource/resource.go

```diff
@@ -29,6 +30,7 @@ import (
 	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
 	"github.com/spf13/cast"
+	"go.mongodb.org/atlas-sdk/v20230201008/admin"
```

Comment:
setting one common alias across the repository will help with updates across the board.
In CLI we using:
```suggestion
	atlasv2 "go.mongodb.org/atlas-sdk/v20230201008/admin"
```

Reply:
It is just less generic name than admin.

Reply:
to be honest I prefer to leave as it is without an alias, I think there is tool to upgrade versions and imports automatically

Reply:
Thanks for the comments here. Just curious, if our integrations are leaning towards a common alias which is more representative, would it be worth renaming the sdk package directly?

Reply:
> would it be worth renaming the sdk package directly?

+1 to this. I found that admin is not very intuitive to quickly be understood for someone new to the code, so I don't know the implications of changign the sdk package name but that would be ideal.

In the meantime we could consider align on an alias common to the code bases we maintain.

@maastha even though this is a nit you might be interested in it because of [this](https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/713)

Reply:
Thanks @marcosuma I align with `atlasv2` as well as I don't find `admin` readable/intuitive until we come up with a plan to rename the sdk package

Reply:
adjusting changes to use atlasv2 alias